Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -2,9 +2,11 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Microsoft.MixedReality.Toolkit.Core.Definitions;
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
 using Microsoft.MixedReality.Toolkit.Core.Extensions;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.BoundarySystem;
+//using Microsoft.MixedReality.Toolkit.Core.Interfaces.DataProviders.SpatialObservers;
```

Comment:
Presumably, this wont be needed Systems which require data providers would be responsible for initiating load correct? That would also potentially allow them to set additional requirements (ex; implemented interfaces).

Reply:
> This namespace is here so we can iterate through the spatial awareness service's data providers.

It may likely that data providers do not require any additional methods. The way that #3180 is reworking the observers, that will be true, We can leave the comment here until that is completed.